examples/notebooks/voltage-regulator/publish_waveforms.ipynb

```diff
@@ -12,48 +12,118 @@
     "operator = Operator(operator_name=\"James Bowery\", role=\"Test Architect\")\n",
-    "operator_id = client.create_operator(operator)\n",
+    "client.create_operator(operator)\n",
+    "alias_james_bowery = client.create_alias(ALIAS_JAMES_BOWERY, operator)\n",
     "\n",
+    "# Create alias for Test Station \"TestStation_12\"\n",
     "test_station = TestStation(test_station_name=\"TestStation_12\")\n",
-    "test_station_id = client.create_test_station(test_station)\n",
+    "client.create_test_station(test_station)\n",
+    "alias_test_station_12 = client.create_alias(ALIAS_TEST_STATION_12, test_station)\n",
+    "\n",
+    "# Create alias for UUT instance \"A861-42367\"\n",
+    "uut = Uut(model_name=\"NI-6508\", family=\"Digital\")\n",
+    "uut_id = client.create_uut(uut)\n",
+    "uut_instance = UutInstance(uut_id=uut_id, serial_number=\"A861-42367\")\n",
+    "client.create_uut_instance(uut_instance)\n",
+    "alias_uut_a861_42367 = client.create_alias(ALIAS_UUT_A861_42367, uut_instance)\n",
+    "\n",
+    "# Create aliases for SoftwareItems\n",
+    "software_item = SoftwareItem(product=\"Windows\", version=\"10.0.19044\")\n",
+    "client.create_software_item(software_item)\n",
+    "alias_software_windows_10 = client.create_alias(ALIAS_SOFTWARE_ITEM_WINDOWS_10, software_item)\n",
+    "software_item_2 = SoftwareItem(product=\"Python\", version=\"3.12\")\n",
+    "client.create_software_item(software_item_2)\n",
+    "alias_software_python_3_12 = client.create_alias(ALIAS_SOFTWARE_ITEM_PYTHON_3_12, software_item_2)\n",
+    "\n",
+    "# Create aliases for HardwareItem\n",
+    "scope = HardwareItem(\n",
+    "    manufacturer=\"NI\",\n",
+    "    model=\"PXIe-5171\",\n",
+    "    serial_number=\"1933B4E\",\n",
+    ")\n",
+    "client.create_hardware_item(scope)\n",
+    "alias_scope = client.create_alias(ALIAS_HARDWARE_ITEM_PXIe_5171, scope)\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db8657b1",
+   "metadata": {},
+   "source": [
+    "## Set up the test result (each test run)\n",
+    "\n",
+    "This setup would be done on each test run. Each time we're going to run a test, we create a `TestResult` using the aliases created for the system from the step above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5aaa09bb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ni.datastore import TestResult\n",
     "\n",
     "test_result = TestResult(\n",
+    "    uut_instance_id=alias_uut_a861_42367.target_id,\n",
+    "    operator_id=alias_james_bowery.target_id,\n",
+    "    test_station_id=alias_test_station_12.target_id,\n",
+    "    software_item_ids=[\n",
+    "        alias_software_windows_10.target_id,\n",
+    "        alias_software_python_3_12.target_id,\n",
+    "    ],\n",
+    "    hardware_item_ids=[alias_scope.target_id],\n",
     "    test_result_name=\"voltage regulator waveforms\",\n",
```

Comment:
Use the string constants for the alias. The operator id can take a GUID (alias_operator.target_id) or the alias (ALIAS_JAMES_BOWERY).

``` python
uut_instance_id = client.create_uut_instance(...)

test_result = TestResult(
    test_result_name="voltage regulator waveforms",
    uut_instance_id=uut_instance_id,
    operator_id=ALIAS_JAMES_BOWERY,
    test_station_id=ALIAS_TEST_STATION_12,
    software_item_ids=[
        ALIAS_SOFTWARE_ITEM_WINDOWS_10,
        ALIAS_SOFTWARE_ITEM_PYTHON_3_12,
    ],
    hardware_item_ids=[ALIAS_HARDWARE_ITEM_PXIe_5171],
)
test_result_id = client.create_test_result(test_result)
```